tests/integration/standard/test_metadata.py

```diff
@@ -1677,11 +1677,12 @@ def test_function_cql_called_on_null(self):
             self.assertRegexpMatches(fn_meta.as_cql_query(), "CREATE FUNCTION.*\) RETURNS NULL ON NULL INPUT RETURNS .*")
 
 
-@unittest.skip('Failing with scylla')
 class AggregateMetadata(FunctionTest):
 
     @classmethod
     def setup_class(cls):
+        if SCYLLA_VERSION:
+            raise unittest.SkipTest("Failing with scylla")
```

Comment:
I don't want any skip to be hard coded.
Not like this and not with unittest.skip.
We skip them only through the yaml file patch inside the python-driver-matrix after explaining why they fail with Scylla and what is the issue number for it.